index.html

```diff
@@ -0,0 +1,170 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="./styles.css" />
+    <title>Exquisite Residence</title>
+    <script
+      src="https://kit.fontawesome.com/e1fe8e6842.js"
+      crossorigin="anonymous"
+    ></script>
+  </head>
+  <body>
+    <header>
+      <nav>
+        <ul class="icon">
+          <li>
+            <a href="#"
+              ><img class="social_icon_f" src="./images/facebook1.jpg" alt=""
+            /></a>
+          </li>
+          <li>
+            <a href="#"
+              ><img class="social_icon_t" src="./images/twitter.png" alt=""
+            /></a>
+          </li>
+          <li>English</li>
+          <li>My Page</li>
+          <li>Layout</li>
+        </ul>
+        <div class="nav_main">
+          <a href="./index.html"
+            ><img class="logo" src="./images/logo.png" alt="#"
+          /></a>
+          <ul class="nav_menu">
+            <li class="nav_menu_item close_button">&times;</li>
+            <li class="nav_menu_item">
+              <a class="other_item link" href="./index.html">Home</a>
+            </li>
+            <li class="nav_menu_item">
+              <a class="other_item" href="./about.html">About</a>
+            </li>
+            <li class="nav_menu_item">
+              <a class="other_item" href="#">Program</a>
+            </li>
+            <li class="nav_menu_item">
+              <a class="other_item" href="#">Join</a>
+            </li>
+            <li class="nav_menu_item">
+              <a class="other_item" href="#">Sponsor</a>
+            </li>
+            <li class="nav_menu_item">
+              <a class="other_item" href="#">News</a>
+            </li>
+            <li class="nav_menu_item">
+              <a class="other_item er_link" href="#">ER Exhibition</a>
+            </li>
+          </ul>
+          <div class="hamburger">
+            <img src="./images/hamburger-menu.png" alt="hambuger" />
+          </div>
+        </div>
+      </nav>
+    </header>
+    <main class="main">
+      <h3 class="main_sub_title">"Homes For the Masses"</h3>
+      <h1 class="main_title">AFFORDTABLE HOUSING EXHIBITION 2022</h1>
+      <p class="main_intro">
+        A sustainable housing delivery system with emphasis on easy access to
+        home ownership for the Nigeria Poeple in an environment where basic
+        infrastructural amenities are readily avaliable.
+      </p>
+      <p class="date">2022.01.16(MON) ~ 18(WED)</p>
+      <p class="main_venue">&copy; Liberty Conference Hall, Auja-Nigeria</p>
+    </main>
+    <section class="activity">
+      <h3 class="activity_title">Main Activities</h3>
+      <div class="bar home_bar"></div>
+      <ul class="activity_menu">

```

Comment:
- [ ] You should use the font color in the design for the activity heading as suggested by previous reviewer because the color you have used has accessibility issue. I suggest you use the secondary font-color given in the design - which is #ec5242.
- [ ] The Iconography you have used are bigger than the ones in the design. Kindly rectify as suggested by previous reviewer.
- [ ] The font-size for the description text is smaller and the color is not white. Kindly rectify as suggested by previous reviewer.